src/main/java/com/google/gcloud/storage/StorageService.java

```diff
@@ -412,14 +420,14 @@ public static Builder builder() {
   Blob create(Blob blob, byte[] content, BlobTargetOption... options);
 
   /**
-   * Return the requested bucket.
+   * Return the requested bucket or {@code null} if not found.
```

Comment:
I am not sure that this is better. Actually, this change was explicitly changing it from getting an exception to null which I think is more user-friendly for these specific methods. I explicitly mentioned that in the review docs and would like to get more input on it. Regardless, if not null I don't think this method should produce any other exception than StorageServiceException with a 404 error code (as it was before).